package.json

```diff
@@ -56,6 +56,7 @@
     "webpack-cli": "3.1.2"
   },
   "dependencies": {
+    "jwt-decode": "^2.2.0",
```

Comment:
can you lock the version pls

Reply:
I use only the base function of it. This locks it to the next patch version, and the `package-lock` file is there to prevent any issue.